enarxbot.py

```diff
@@ -117,3 +118,17 @@ def create_card(column, content_id, content_type):
         if error["resource"] != "ProjectCard" or error["code"] != "unprocessable":
             raise
         print("Card already in project.")
+
+# Get all issue numbers related to a PR.
+def get_related_issues(pr):
+    # Regex to pick out closing keywords.
+    regex = re.compile("(close[sd]?|fix|fixe[sd]?|resolve[sd]?)\s*:?\s+#(\d+)", re.I)
```

Comment:
What about cases where we'd close multiple issues? Does the bot need to take care of that?